go.mod

```diff
@@ -11,11 +11,11 @@ require (
 	github.com/containerd/containerd v1.6.3
 	github.com/containerd/continuity v0.3.0
 	github.com/containerd/go-cni v1.1.5
-	github.com/containerd/imgcrypt v1.1.4
+	github.com/containerd/imgcrypt v1.1.5-0.20220421044638-8ba028dca028
```

Comment:
Why do we need updating imgcrypt and typeurl?

Reply:
I'm not sure, I just replaced containerd.